pkgs/top-level/all-packages.nix

```diff
@@ -1918,7 +1918,9 @@ in
 
   mpdcron = callPackage ../tools/audio/mpdcron { };
 
-  mpdris2 = callPackage ../tools/audio/mpdris2 { };
+  mpdris2 = callPackage ../tools/audio/mpdris2 {
+    pythonPackages = python3Packages;
```

Comment:
```suggestion
```

Reply:
I ended up mimicking what the nixpkgs manual shows and what other python applications in nixpkgs do.